content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,431 @@
+-->
+### 将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io
+[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 PR 已经被合并。
+更多细节参考 [wiki 页面](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\))，
+同时[公告](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ)已发送到 kubernetes 开发邮件列表。
+
+<!--
+### Promoted SeccompDefault to Beta
+
+SeccompDefault promoted to beta, see the tutorial [Restrict a Container's Syscalls with seccomp](https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads) for more details.
+-->
+### SeccompDefault 升级为 Beta
+
+SeccompDefault 升级为 Beta，
+更多细节参考教程[用 seccomp 限制一个容器的系统调用](/zh-cn/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads)。
+
+<!--
+### Promoted endPort in Network Policy to Stable
+
+Promoted `endPort` in [Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports) to GA. Network Policy providers that support `endPort` field now can use it to specify a range of ports to apply a Network Policy. Previously, each Network Policy could only target a single port.
+
+Please be aware that `endPort` field **must be supported** by the Network Policy provider. If your provider does not support `endPort`, and this field is specified in a Network Policy, the Network Policy will be created covering only the port field (single port).
+-->
+### 网络策略中 endPort 升级为 Stable
+
+[网络策略](/zh-cn/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports)中的 
+`endPort` 已经迎来 GA 通用版。
+支持 `endPort` 字段的网络策略提供程序，现在可使用该字段来指定端口范围来应用网络策略。
+在之前的版本中，每个网络策略只能指向单一端口。
+
+请注意，网络策略提供程序 **必须支持** `endPort` 字段。
+如果提供程序不支持 `endPort`，又在网络策略中指定了此字段，
+则会创建出仅覆盖端口字段（单端口）的网络策略。
+
+<!--
+### Promoted Local Ephemeral Storage Capacity Isolation to Stable
+The [Local Ephemeral Storage Capacity Isolation](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/361-local-ephemeral-storage-isolation) feature moved to GA. This was introduced as alpha in 1.8, moved to beta in 1.10, and it is now a stable feature. It provides support for capacity isolation of local ephemeral storage between pods, such as `EmptyDir`, so that a pod can be hard limited in its consumption of shared resources by evicting Pods if its consumption of local ephemeral storage exceeds that limit.
+-->
+### 本地临时容器存储容量隔离升级为 Stable
+[本地临时存储容量隔离功能](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/361-local-ephemeral-storage-isolation)已经迎来 GA 通用版。
```

Comment:
```suggestion
[本地临时存储容量隔离功能](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/361-local-ephemeral-storage-isolation)已经迎来 GA 正式发布版本。
```

Reply:
Thank you for your suggestion, it has been amended.

Reply:
> Thank you for your suggestion, it has been amended.

If you have modified all, please click the `Resolve conversation` button.